db/migrate/20200603115449_add_last_activity_at_to_needs.rb

```diff
@@ -0,0 +1,14 @@
+class AddLastActivityAtToNeeds < ActiveRecord::Migration[6.0]
+  def up
+    add_column :needs, :last_activity_at, :timestamp, default: -> { 'NOW()' }, null: false
+
+    Need.all.each do |need|
+      last_activity = need.matches.pluck(:updated_at).max || need.updated_at
+      need.update(last_activity_at: last_activity)
+    end
+  end
+
+  def down
```

Comment:
le `down` est nécessairement séparé? Je trouve ça plus clair quand on fait un `change`, quitte à mettre un bloc `up_only` dedans.

(J’y tiens pas plus que ça, c’est juste par curiosité.)

Reply:
en fait j'ai appris avec le up and down donc c'est a ça que je pense en premier